src/stories/Responses/UserCanAddResponse/story.js

```diff
@@ -58,7 +64,27 @@ const validateInput = async (prepareResult) => {
 const handle = async ({ prepareResult, authorizeResult }) => {
   try {
     await validateInput(prepareResult);
-    return ResponseRepo.create(prepareResult);
+    let res = await ResponseRepo.create(prepareResult);
+    console.log("res>>>>>", res, JSON.stringify(res));
```

Comment:
@Subbu-betalectic Remove the console logs

Reply:
Removed the console logs